### ISSUE TITLE
harbor upgrade not blocked from 1.8.0 to 2.0.0

### ISSUE THREAD

**Issue by bitsf** [goharbor/harbor#11223]
user can directly use harbor 2.0.0 installer to upgrade from 1.8.0
![image](https://user-images.githubusercontent.com/188115/77398442-7d460a00-6de2-11ea-8f41-6d47afc21457.png)

only the harbor-core container failed to startup with error
![image](https://user-images.githubusercontent.com/188115/77398090-c77abb80-6de1-11ea-9c30-b8d2dc991b91.png)

This should be a bad experience, that we should consider block this when run installer

**Comment by reasonerjt** [comment]
we can mitigate the issue by fail early in the conf migration.

**Comment by ninjadq** [comment]
fixed in #11381

**Comment by bitsf** [comment]
user need to run migrator to get this blocker notify.
```
docker run -it --rm -v `pwd`/harbor.yml:/harbor-migration/harbor-cfg/harbor.yml goharbor/prepare:v2.0.0 migrate -i /harbor-migration/harbor-cfg/harbor.yml
```
will got warning
```
the version 1.8.0 not supported, make sure the version in input file above 1.8.0
```
